src/test/groovy/org/gradle/profiler/AbstractProfilerIntegrationTest.groovy

```diff
@@ -8,18 +10,40 @@ abstract class AbstractProfilerIntegrationTest extends AbstractIntegrationTest {
 
     private static int NUMBER_OF_HEADERS = 4
 
+    static List<String> supportedOnCurrentJvm(List<String> gradleVersions) {
```

Comment:
How about calling this `gradleVersionsSupportedOnCurrentJvm()`?

Reply:
Renamed.